### DISCUSSION TITLE
Homebrew installation erroe

### DISCUSSION THREAD

**Discussion by DavidAdeboye** [Homebrew/discussions#5769] [Everyday usage] [answered]
### Output of `brew config`

```shell
fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed
```

### Output of `brew doctor`

```shell
fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed
```

### Description of issue

Been trying to install homebrew from /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"   and i keep getting this errors  
fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed
![Screenshot 2024-11-26 025502](https://github.com/user-attachments/assets/cbf53dec-22e4-4a1a-afd7-def928a47195)
![Uploading Screenshot 2024-11-26 025502.png…]()

**Comment by SMillerDev** [comment, marked as answer]
Duplicate of #666
